### PR TITLE
64 feature article  card

### PR DESCRIPTION
**Pull request by AndyBowerman**
Created article card. Contains placeholder text and images to be replaced by state at a later date.
Styling applied based on figma design. Will need to be updated with the project variables once they are available.
Currently passing all tests written. Tested elements of the component appear on the users page.

### CODE REVIEW

File: src/components/ArticleListCards/ArticleListCards.test.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { render, screen} from "@testing-library/react";
+import ArticleListCards from "./ArticleListCards";
+import ""
```
Comment (nologyRob) [inline src/components/ArticleListCards/ArticleListCards.test.jsx:3]: What's this?

File: src/components/ArticleListCards/ArticleListCards.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
```
Comment (nologyRob) [inline src/components/ArticleListCards/ArticleListCards.jsx:1]: Rewrite the entire component with props in mind.

File: src/components/ArticleListCards/ArticleListCards.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        .card__title {
+            font-weight: $font-weight-regular;
+            font-size: $font-size-medium;
+            line-height: 32px;
+            color: $color-black;
+        }
+
+    }
+
+
```
Comment (nologyRob) [inline src/components/ArticleListCards/ArticleListCards.scss:39]: Sorry but remove extra line

File: src/components/ArticleListCards/ArticleListCards.scss [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        line-height: 24px;
+        color: $color-teal;
+    }
+
+    &__links {
+        font-weight: $font-weight-regular;
+        font-size: $font-size-extra-small;
+        line-height: 20px;
+    }
+
```
Comment (nologyRob) [inline src/components/ArticleListCards/ArticleListCards.scss:52]: Sorry but remove extra line

**Review by nologyRob — CHANGES_REQUESTED** [review]

**Review by AdrianRotila — CHANGES_REQUESTED** [review]

**Pull request closed without merging**